core/modules/filter/filter.pages.inc

```diff
@@ -461,9 +461,7 @@ function filter_format_editor_link_form_submit($form, &$form_state) {
     }
   }
   else {
-    // Modify references to other URIs if not an external link.
-    $url = (substr($uri, 0, 4) == 'http') ? $uri : substr(base_path(), 0, -1) . $uri;
-    $form_state['values']['attributes']['href'] = $url;
+    $form_state['values']['attributes']['href'] = $uri;
```

Comment:
Rather than never adding the base path, we should check to see if the person (or autocpmplete) providing the link included it, or not, and add it only if it is not already present.

Reply:
I'm not sure I understand. The base path is always added. I took this out because this adds it a second time.   

Please read the [comment (and subsequent)](https://github.com/backdrop/backdrop-issues/issues/5886#issuecomment-1345147356) that explain this in the main conversation.